src/client.ts

```diff
@@ -151,4 +153,55 @@ export class Client {
 
     return new TaskInfo(message, state, options.processAt);
   }
+
+  async bulkEnqueue(
+    tasks: Task[],
+    opts?: Partial<Options>
+  ): Promise<TaskInfo[]> {
+    const taskInfoList: TaskInfo[] = [];
+    const pendingMsgList: TaskMessage[] = [];
+    const scheduledMsgList: TaskMessage[] = [];
+
+    let options: Options = { ...DEFAULT_OPTIONS, ...opts };
+
+    for (const task of tasks) {
+      options = { ...options, ...task.opts };
```

Comment:
Current order of `options` spreading results in Task-defined options having highest precedence which is different than current `enqueue` order. Will probably want to standardize queue-level options having greatest precedence:

```suggestion
    let options: Options = { ...DEFAULT_OPTIONS };

    for (const task of tasks) {
      options = { ...options, ...task.opts, ...opts };
```